docs/developer/contributing/development-functional-tests.asciidoc

```diff
@@ -84,7 +84,7 @@ export TEST_BROWSER_HEADLESS=1
 export TEST_THROTTLE_NETWORK=1
 ----------
 
-** When running against a Cloud deployment, some tests are not applicable. To skip tests that do not apply, use --exclude-tag.  An example shell file can be found at: {blob}test/scripts/jenkins_cloud.sh[test/scripts/jenkins_cloud.sh]
+** When running against a Cloud deployment, some tests are not applicable. To skip tests that do not apply, use --exclude-tag.  An example shell file can be found at link:{kib-repo}/tree/{branch}/test/scripts/jenkins_cloud.sh[test/scripts/jenkins_cloud.sh]
```

Comment:
Same as above, the resulting link is currently https://github.com/elastic/kibana//tree/7.10/test/scripts/jenkins_cloud.sh

```suggestion
** When running against a Cloud deployment, some tests are not applicable. To skip tests that do not apply, use --exclude-tag.  An example shell file can be found at link:{kib-repo}/tree/{branch}test/scripts/jenkins_cloud.sh[test/scripts/jenkins_cloud.sh]
```